app/views/rooms/show.html.erb

```diff
@@ -0,0 +1,48 @@
+<% content_for :header do %>
+  <%= render "layouts/header" %>
+<% end %>
+
+<div id="room-id" data-room-id="<%= @room_id %>">
+
+  <div class="container whiteboard">
+
+    <h1>Aktueller Chatroom: <%= @selectedRoom.name %></h1>
+    <br>
+    <hr>
+    <br>
+
+
+    <ul id="messages_container">
+      <!-- loading existing messages when entering the chatroom -->
+      <% if @roomMessages.present? %>
```

Comment:
In Rails werden alle Variablennamen und Symbole im `snake_case` Style geschrieben.